src/core/contract_address/sierra_contract_address.rs

```diff
@@ -142,3 +151,36 @@ mod tests {
         )
     }
 }
+
+struct PythonJsonFormatter;
+
+impl Formatter for PythonJsonFormatter {
```

Comment:
Wouldn't this also need to implement [to_write_string_fragment](https://github.com/xJonathanLEI/serde_json_pythonic/blob/patch/pythonic.patch#L96-L116)?

Also, I think the heart of the issue is to avoid all this as well? This was the status quo before `serde_json_pythonic`.

Reply:
Why would `PythonJsonFormatter` need to implement `to_write_string_fragment`?

You mean avoiding `serde_json` at all? Not just `serde_json_pythonic`?

Reply:
the goal of the ticket was just to remove serde_json_pythonic usage and just use serde_json.
So I think it is ok to implement it like that.

Reply:
I understand now the need to implement `write_string_fragment()` and I've implemented it. I've also fixed a bug I found on the other implementation where, for example, the unicode character `0` would be printed as `\u   0` instead of `\u0000`.